utils/solana.commands.helpers.ts

```diff
@@ -63,6 +102,104 @@ export const keypairFromPrivateKey = (
   }
 };
 
+export const getKeypair = async ({
+  name,
+  mnemonic,
+  privateKey,
+}: {
+  mnemonic: string | undefined;
+  name: string | undefined;
+  privateKey: string | undefined;
+}) => {
+  let keypair: anchor.web3.Keypair;
+  if (privateKey) {
+    keypair = keypairFromPrivateKey(privateKey);
+  } else if (mnemonic) {
+    keypair = await keypairFromMnemonic(mnemonic);
+  } else if (name) {
+    const privateKey = getAccountData<SolanaAccountData>(
+      "solana",
+      name
+    )?.privateKey;
+    keypair = keypairFromPrivateKey(privateKey!);
```

Comment:
We shouldn't use the non-null assertion operator (in fact, we should add a config rule to prevent its usage), `privateKey` uses optional chaining so it very well could be `undefined`. Thankfully the `keypairFromPrivateKey` function handles this case gracefully with its own try/catch, but we should still not resort to that.

Reply:
I've added explicit private key undefined check following the same approach as you used in `setupEvmTransaction`:

https://github.com/zeta-chain/toolkit/blob/958a4db547e43b19c34224c473d4135370d546a1/utils/evm.command.helpers.ts#L54-L66

759727f

If you think we should approach this differently, please, feel free to update the code 🙏

Reply:
Actually, I had to remove this refine rule, because we're using default value for name, so it's always going to throw an error assuming we passed both name and mnemonic/pk. 99351b3

Reply:
Commander's `conflicts` still catches this, so it shouldn't be a problem.